DeepSpeech.py

```diff
@@ -543,20 +579,39 @@ def train():
             session.run(init_op)
             loaded = True
 
-        tfv1.get_default_graph().finalize()
```

Comment:
Please re-add this line after the block below, when the final graph modifications have been made.